### PR TITLE
Add announcement for 2.4

### PR DESCRIPTION
**Pull request by parisk**

### CODE REVIEW

File: _posts/2017-03-08-xtermjs-2.4.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+excerpt: Xterm.js 2.4 just landed with the new WebLinks feature!
+---
+
+In xterm.js 2.4 we are introducing a great new feature, improvements and fixes!
+
+First off, let's start with what's new!
+
+## 🆕  WebLinks
+
+Now xterm.js will detect and WebLinks in the terminal's buffer and convert them to `a` elements, which you can click to navigate to them (all link elements open in a new browser tab by default).
```
Comment (Tyriar) [inline _posts/2017-03-08-xtermjs-2.4.md]: which you can click to navigate to them
->
which you can click to follow

File: _posts/2017-03-08-xtermjs-2.4.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+author_twitter: pariskasid
+categories: xtermjs update
+excerpt: Xterm.js 2.4 just landed with the new WebLinks feature!
+---
+
+In xterm.js 2.4 we are introducing a great new feature, improvements and fixes!
+
+First off, let's start with what's new!
+
+## 🆕  WebLinks
```
Comment (Tyriar) [inline _posts/2017-03-08-xtermjs-2.4.md]: Separating words with space would probably be better

File: _posts/2017-03-08-xtermjs-2.4.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+Now xterm.js will detect and WebLinks in the terminal's buffer and convert them to `a` elements, which you can click to navigate to them (all link elements open in a new browser tab by default).
+
+![image](https://cloud.githubusercontent.com/assets/1188592/23652411/bff2c79c-0320-11e7-9df8-fd0eea93e646.png)
+
+For more information on this take a look a the corresponding Pull Request: #538.
```
Comment (Tyriar) [inline _posts/2017-03-08-xtermjs-2.4.md]: take a look a
->
take a look at

File: _posts/2017-03-08-xtermjs-2.4.md (hunk 2 above)
Comment (parisk) [inline _posts/2017-03-08-xtermjs-2.4.md]: I named it this way so we can refer to this feature with one word. It's just a naming we can change later on.

**Pull request merged**